lib/scenic/adapters/postgres.rb

```diff
@@ -166,6 +171,16 @@ def try_index_create(index)
         connection.execute("ROLLBACK TO SAVEPOINT #{index.index_name}")
         "index '#{index.index_name}' on '#{index.object_name}' is no longer valid and has been dropped."
       end
+
+      def version_number
```

Comment:
My change to make `postgresql_version` public in Rails 5 was merged. I'm tempted to make this `connection.send(:postgresql_version)`. Or maybe even check if the method exists as public, call that if it does, otherwise use send.